### ISSUE TITLE
[Quests issues]: ForceCraft Time Torch

### ISSUE THREAD

**Issue by ttryy** [AllTheMods/All-the-mods-9-Sky#517]
### Modpack Version

1.0.7

### Describe the issue

![image](https://github.com/AllTheMods/All-the-mods-9-Sky/assets/8318976/6bcc73b2-5f8c-4def-a8ec-6125cb423f3a)

Incorrect information: the Time Torch is usable and activated.

**Comment by lpacphantom** [comment]
You were able to tier up to 6? For me, the healing infusion isn't working, so I'm stuck at 5. Force Rod + ghast tear isn't valid for some reason.
I've had some odd issues with recipes in this pack. There was a time when I loaded up and Infused Alloy from Mekanism wasn't working, but I'd used it previously and the next time I started it up, it was working again. I was hoping that was the case for this too, but restarting unfortunately didn't fix it.
